packages/graph/deployment/postgres/init-user-db.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+
+psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" --dbname "$POSTGRES_DB" <<-EOSQL
+  CREATE USER $HASH_GRAPH_PG_USER WITH PASSWORD '$HASH_GRAPH_PG_PASSWORD';
+  CREATE DATABASE $HASH_GRAPH_PG_DATABASE;
+  GRANT ALL PRIVILEGES ON DATABASE $HASH_GRAPH_PG_DATABASE TO $HASH_GRAPH_PG_USER;
```

Comment:
Do we want to limit privileges to CRUD?

Reply:
Have addressed this in https://github.com/hashintel/hash/pull/862/commits/bc5ae61eb0e9e7fbf903ab82327f1b0c216160c0